packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -148,6 +149,55 @@ export default function NewCommentModal({
     onInputChange?.(input);
   }, [input]);
 
+  // return (
+  //   <Modal
+  //     contentRef={modalRef}
+  //     onRequestClose={confirmClose}
+  //     kind={Modal.Kind.FlexibleCenter}
+  //     size={Modal.Size.Small}
+  //     tabs={['Write', 'Preview']}
+  //     {...props}
+  //   >
+  //     <Modal.Header />
+  //     <Modal.Body tab="Write">
```

Comment:
If we pass a tab to Modal.Body or Modal.Footer, it will only render contents if the active tab matches.